HW0/P4/P4.py

```diff
@@ -29,48 +29,123 @@
     #
     # Load true trajectory and plot it
     # Normally, this data wouldn't be available in the real world
+    s_true = np.loadtxt('P4_trajectory.txt', delimiter=',')
+    x_coords = s_true[:,0]
+    y_coords = s_true[:,1]
+    z_coords = s_true[:,2]
     #####################
 
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '--b', label='True trajectory')
+    ax.plot(x_coords, y_coords, z_coords,
+            '--b', label='True trajectory')
 
     #####################
     # Part 2:
     #
     # Read the observation array and plot it (Part 2)
-    #####################
-
-    # ax.plot(x_coords, y_coords, z_coords,
-    #         '.g', label='Observed trajectory')
+    s_measure = np.loadtxt('P4_measurements.txt', delimiter=',')
+    
+    V = np.matrix([[1/rx, 0, 0], [0, 1/ry, 0], [0, 0, 1/rz]])
```

Comment:
Another option could be np.diag(np.array()), so you don't have to type all those 0's :)